samples/linK-2/input.dat

```diff
@@ -21,11 +21,11 @@ molecule mol {
 }
 
 set {
-    scf_type dfdirj+link
+    scf_type dfdirj+snlink
```

Comment:
This is `samples/` editing again which are autogenerated from `tests/`. Fine to leave it in, but it'll get overwritten soon-ish.

Reply:
Yeah, that's fine! I tend to edit the `samples/` files a lot for testing as well, which is where these changes come from. Any pushing of these files to GitHub is on oversight, and I have no qualms about them being overwritten.